CHANGELOG.md

```diff
@@ -22,6 +22,8 @@ Dropping a requirement of a major version of a dependency is a new contract.
 
 ## [Unreleased]
 [Unreleased]: https://github.com/atlassian/docker-infrastructure/compare/release-0.3.2...master
+### Changed
```

Comment:
Oops, I missed this detail.
`Changed` suggests breaking a contract. Ubuntu version was not a behavioral contract. `AbstractJiraFormula` does not specify the Ubuntu version. Currently this is a transparent change: `Fixed`.
If we want callers to be able to rely on the Ubuntu version, then we gotta add it to the Javadoc. Then it'd become  a new contract: `Added`.

Reply:
Thanks. Let's stick to not extending the contract hence treating it as `Fixed`.